src/pages/patientView/copyNumberAlterations/column/CustomDriverColumnFormatter.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { DiscreteCopyNumberData } from 'cbioportal-ts-api-client';
+import _ from 'lodash';
+import { AnnotatedNumericGeneMolecularData } from 'shared/model/AnnotatedNumericGeneMolecularData';
+
+/**
+ * Discrete Copy Number Formatter.
+ */
+export default class CustomDriverColumnFormatter {
```

Comment:
maybe add CopyNumber to this name since it won't be apparent it's specific to CN.

Reply:
There are other column formatters in mutation, fusions, and CN table, that do not have this specific name. Do you want to update all of them as well?